includes/class-wc-payments-account.php

```diff
@@ -330,6 +331,46 @@ public function get_fraud_services_config() {
 		return $filtered_services_config;
 	}
 
+	/**
+	 * Checks if the request is for the Capital view offer redirection page, and redirects to the offer if so.
+	 *
+	 * Only admins are be able to perform this action. The redirect doesn't happen if the request is an AJAX request.
+	 * This method will end execution after the redirect if the user requests and is allowed to view the loan offer.
+	 */
+	public function maybe_redirect_to_capital_offer() {
+		if ( wp_doing_ajax() ) {
+			return;
+		}
+
+		// Safety check to prevent non-admin users to be redirected to the view offer page.
+		if ( ! current_user_can( 'manage_options' ) ) {
```

Comment:
Should we use `manage_woocommerce` here instead?

Reply:
Is there a chance that one user is a WP-admin, but not a WC-admin?

Reply:
I think so. Also, all other checks on the plugin are using `manage_woocommerce`, so I changed it to that in https://github.com/Automattic/woocommerce-payments/pull/3673/commits/19ea8d421db7d0418bd3ac47cb47b1c0fe3e97fd.